### ISSUE TITLE
Running Installer Not Working

### ISSUE THREAD

**Issue by dominikfindeisen1** [OfficialDonut/VisualBukkit#78]
When i open the "Visual Bukkit Installer" file, it shows a small window with loading bar, then it disappears and on the bottom bar shows an icon of lava bucket, that is an empty window (fully transparent)
<img width="898" alt="Zrzut ekranu 2023-02-11 o 11 02 46" src="https://user-images.githubusercontent.com/96972153/218252168-dd590748-4b0d-4150-a516-50e0a238b178.png">
<img width="898" alt="Zrzut ekranu 2023-02-11 o 11 02 46" src="https://user-images.githubusercontent.com/96972153/218252205-df36e6ef-e115-43d0-a1fa-a6501e1ed41c.png">
<img width="898" alt="Zrzut ekranu 2023-02-11 o 11 03 13" src="https://user-images.githubusercontent.com/96972153/218252206-84af9a29-e102-4f1e-99e8-0694473f0da2.png">

**Comment by dominikfindeisen1** [comment]
Sorry, one screenshot uploded twice

**Comment by time-attack** [comment]
Are you using an m1 mac?

**Comment by pixityy** [comment]
same here, hopefully somebody can resolve this in the future!

**Comment by StarDroid77** [comment]
Yeah its very frustrating, now I have to either bug my brother for his windows computer or tell my friend exactly what to code over a discord call with screen share.

and I am also using an m1 mac
